skimage/feature/tests/test_peak.py

```diff
@@ -351,57 +355,35 @@ def test_four_quadrants(self):
                                          exclude_border=False)
         assert np.all(result == expected)
 
-    def test_disk(self):
```

Comment:
Why is this test removed?

Reply:
`footprint` no more overwrites `min_distance`. For this test to make sens, we need to set `min_distance` to 0.